examples/tests/invalid/access_modifiers.w

```diff
@@ -0,0 +1,100 @@
+class Foo {
+  protected protected_field: str;
+  private_field: str;
+  public public_field: str;
```

Comment:
Can access modifiers be added to an interface? (add a valid / invalid unit test)

Reply:
Good callout. Anything implementing an interface must be `public`. I'll need to added verification and tests...

Reply:
https://github.com/winglang/wing/pull/4133/commits/d3d1c3c33f35fb43ed570f2881d5b8782821d5fa